### PR TITLE
chore: Document access token to read from github packages

### LINKED ISSUE #51: Document access token to read from github packages

### PR DESCRIPTION
**Pull request by fqutishat**
closes #51

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: docs/demo/build.md [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
 ## Developer Setup
-In order to access edge service image and user agent image you have to create personal token with read:packages and repo permissions ([personal token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line)).
+In order to access edge service image and user agent image you have login to docker.pkg.github.com.
  
-Run the following command using your newly generated personal token:
+Run the following command using your github username:
 
 ```
-docker login -u <username> -p <github token with read:packages permission> docker.pkg.github.com
+docker login -u <username> -p e438ec2cd264600e6d3fec6dd8aeda1633e4f929 docker.pkg.github.com
```
Comment (troyronda) [inline docs/demo/build.md]: I'm not sure I understand this change. You need to create a GitHub token with these permissions.

File: docs/demo/build.md [hunk 2]
```diff
@@ -4,12 +4,12 @@
 - Go 1.13
 
 ## Developer Setup
-In order to access edge service image and user agent image you have to create personal token with read:packages and repo permissions ([personal token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line)).
+In order to access edge service image and user agent image you have login to docker.pkg.github.com.
  
-Run the following command using your newly generated personal token:
+Run the following command using your github username:
```
Comment (troyronda) [inline docs/demo/build.md]: and your token.

File: docs/demo/build.md [hunk 3]
```diff
@@ -4,12 +4,12 @@
 - Go 1.13
 
 ## Developer Setup
-In order to access edge service image and user agent image you have to create personal token with read:packages and repo permissions ([personal token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line)).
```
Comment (troyronda) [inline docs/demo/build.md]: Isn't the link useful?

File: docs/demo/build.md (hunk 2 above)
Comment (fqutishat) [reply docs/demo/build.md]: you don't need to generate new token

Comment (fqutishat) [reply docs/demo/build.md]: we can use one token to read

**Pull request closed without merging**